plugins/integration/providers/voicebase/lib/kVoicebaseFlowManager.php

```diff
@@ -65,10 +74,10 @@ public function updatedJob(BatchJob $dbBatchJob)
 			$formatsArray[] = "TXT";
 			$contentsArray = $clientHelper->getRemoteTranscripts($entryId, $formatsArray);
 			KalturaLog::debug('contents are - ' . print_r($contentsArray, true));
-			$transcript = $this->getAssetsByLanguage($entryId, array(TranscriptPlugin::getAssetTypeCoreValue(TranscriptAssetType::TRANSCRIPT)), $spokenLanguage, true);
 			$captions = $this->getAssetsByLanguage($entryId, array(CaptionPlugin::getAssetTypeCoreValue(CaptionAssetType::CAPTION)), $spokenLanguage);
-		
-			$this->setObjectContent($transcript, $contentsArray["TXT"], null, true);
+			$accuracy = $clientHelper->calculateAccuracy($entryId);
```

Comment:
@rkreich @shpiNi - why do we need to round at all? I would just cut after two decimals (0.862328 will become 0.86)